.circleci/application.yml

```diff
@@ -1,3 +1,4 @@
 SERVER_URL: 'localhost:3000'
 PASSWORD_RESET_URL: 'http://localhost:3000'
 SPOTS_COUNT_LIMIT: '10'
+PUSH_APP_ID: '8493b49a-da03-4700-8c2e-47d7797472a4'
```

Comment:
This should be an ENV VAR in CircleCi https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project

This is a private key, should not be public.